src/sentry/integrations/msteams/actions/notification.py

```diff
@@ -54,7 +60,16 @@ def send_notification(event, futures):
             ).build_group_card(notification_uuid=notification_uuid)
 
             client = MsTeamsClient(integration)
-            client.send_card(channel, card)
+            with MessagingInteractionEvent(
+                interaction_type=MessagingInteractionType.SEND_ISSUE_ALERT_NOTIFICATION,
+                spec=MsTeamsMessagingSpec(),
+            ).capture() as lifecycle:
+                lifecycle.add_extras({"integration_id": integration.id, "channel": channel})
+                try:
+                    client.send_card(channel, card)
+                except ApiError as e:
+                    lifecycle.add_extras({"integration_id": integration.id, "channel": channel})
```

Comment:
Duplicate lifecycle extras?

Reply:
We probably want to set this at the top like the original PR had so that any non `ApiError` exceptions carry integration and channel information